tests/common/database_fixture.cpp

```diff
@@ -120,6 +120,11 @@ database_fixture::database_fixture()
    {
       options.insert(std::make_pair("max-ops-per-account", boost::program_options::variable_value((uint64_t)75, false)));
    }
+   if (current_test_name == "setting_max_operation_limit_get_account_history_operations")
+   {
+      options.insert(std::make_pair("max-ops-per-account", boost::program_options::variable_value((uint64_t)125, false)));
+      app.set_options_max_account_history_operations_limit(300);
```

Comment:
Why do you set this to 300 here? This value will be overwritten by the options anyway.
Getting rid of this here means you also don't need the setter method at all.

Reply:
The goal was to test whether the FC_ASSERT  could be configured from the config file value.
1) I am able to get the value into the _max_account_history_operations_limit
2) But was not able to test it so I created this setter to set the value and test it via boost_test
3) Reason for putting 300 value was to demonstrate the successful working of testcase scenario
Validating if the value can go above 100 which was hard coded previously 
"1. created 1 account with higher return limits(200)"

I agree that max-ops-per-account would limit this value. But I wanted to verify the testcase of whether the value can go above 100 which was hard coded. So I created the below scenario
"create a bunch of accounts and return set the limit to 125 and the max operation size to higher return limits(125)-- operation is limited by the max-operation-limit tickets:"

Do you think user would need this api scenario. If not I will take it out.

Reply:
Oh, now I see. The option you're setting above doesn't have an effect because neither `application::initialize(options)` nor `application::startup` get called by the test suite.

Please fix the test suite to use the configured option value and remove the `set_options...` method. One way to do this would be to refactor some of the option parsing from `application_impl::startup` into a separate method and call that from both `application_impl::startup` and from the `database_fixture` constructor.

Reply:
Looks good, thanks!